### ISSUE TITLE
Feature Request - Floating button

### ISSUE THREAD

**Issue by DMeredith88** [joecks/clipboard-actions-community#87]
Like the Pocket app

**Comment by joecks** [comment]
Can you explain in detail what functionality you expect from that?

**Comment by DMeredith88** [comment]
Hi joecks!

When a piece of text or thing is copied, or ideally selected, it would be great to display a similar type of button floating near the selected / copied thing.

When pressed the available actions can pop out as other circle buttons. This would be great because then you wouldn't have to go to the notification bar to take your next action, it'd be available to you straight away.

![pocket](https://user-images.githubusercontent.com/32440738/74591946-7faf8a00-5014-11ea-8caa-f82df7eaf73d.png)

**Comment by Ilithy** [comment]
Yes this feature would be very handy, it could replace the notification, which would be great on a tablet (or phones with large screen
